src/components/optionSet/OptionSetStyle.js

```diff
@@ -21,41 +21,46 @@ const addOptionStyle = (option, index) => ({
     },
 })
 
-const OptionSetStyle = ({
-    id,
-    options,
-    optionSet,
-    loadOptionSet,
-    setOptionStyle,
-}) => {
+const OptionSetStyle = ({ styledOptionSet }) => {
```

Comment:
Since redux is already being used in this component (dispatch), I'm wondering if it would make more sense to get styledOptionSet with useSelector directly instead of it being passed through the components as a prop? Really not an important thing.

Reply:
Keeping the current solution as we already have this property in the parent component.